lib/navigation/preprocess_imu.hpp

```diff
@@ -1,19 +1,24 @@
+#pragma once
+
+#include "types.hpp"
+
 #include <array>
 #include <cstdint>
 
 #include "core/types.hpp"
 
 namespace hyped::navigation {
+
 class ImuPreprocessor {
  public:
   ImuPreprocessor();
 
-  core::ImuData processData(const core::RawImuData raw_imu_data);
+  std::optional<ImuData> processData(const core::RawImuData &raw_imu_data);
```

Comment:
This change is already getting implemented in #39. We're in the process of writing preprocessor just now so currently catching stuff like this as we go.

Reply:
merging should be fine then :)